sdk/spring-sdk/src/test/java/kalix/springsdk/testmodels/action/ActionsTestModels.java

```diff
@@ -112,4 +115,25 @@ public Action.Effect<Message> message(@PathVariable String one) {
       return effects().reply(new Message(one));
     }
   }
+
+  public static class StreamOutAction extends Action {
+    @GetMapping("/message")
+    public Flux<Effect<Message>> message() {
+      return Flux.just(effects().reply(new Message("hello")));
+    }
+  }
+
+  public static class StreamInAction extends Action {
+    @PostMapping("/message")
+    public Action.Effect<Message> message(@RequestBody Flux<Message> messages) {
```

Comment:
Did you actually get streaming in to work after all?

Reply:
No, this is being used to test that if someone try to create an action receiving a flux, it will fail to register the action.